index.html

```diff
@@ -70,6 +76,243 @@ <h1>I'm Abdi</h1></div>
+              
+            
+            </div>
+
+      
+            <div class="cards card7">
+                <h1 class="cards-header">Profesional Art <br>Printing Data</h1>
+                <div class="cards_description">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+                        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+              
+                <div class="card_buttons">
+                    <a href="#">  <button type="button" id="button_link">html</button></a>
+                    <a href="#">   <button type="button" id="button_link">bootstrap</button></a>
+                    <a href="#">  <button type="button" id="button_link">Ruby</button></a>
+                </div>
+                <div class="seeProjectCards">
+                    <a href="#"><button type="button" id="button_linkProj">See Project</button></a>  
+              </div>
+            
+              
+            
+            </div>
+           
+
+
+        </div>
+       
+       
+        
+
+
+
+
+
+
+
+    </section>
+    <div class="about_Me">
+        <div class="abtMe_content">
+        <h1 class="abtMe_Header">About me
+            
+        </h1>
+        <div class="abtMedescription">
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <button type="button" class="myresumeButton">Get My Resume</button>
+        <img src="./assets/images/lineForAbtMe.png" id="lineAbtMe">
+
+
+
+
+
+
+        </div>
+
+
+
+
+
+
+
+
+        <div class="experiance">
+        <div id="language">
+           
+            <div id="icon_exprex"> <img src="./assets/images/rombusIcon.png"><h1>Languages</h1></div>
+            <ul>
+                <li>
+                    Javascript
+
+                </li>
+
+
+                <li>Html</li>
+                <li>Css</li>
+                
+            </ul>
+            </div>
+        
+
+            <div id="frameworks" >     <div id="icon_exprex"> <img src="./assets/images/RectangleIcon.png"><h1>Frameworks</h1></div>
+                <ul>
+                <li>
+                    Bootstrap
+
+                </li>
+<li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+</ul>
+            </div>
+
+<div id="skill"> <div id="icon_exprex"> <img src="./assets/images/circleIcon.png"><h1>Skills</h1></div>
+    <ul>
+        <li>
+            Codekit
+
+        </li>
+
+
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li id="theLast">Terminal</li>
+    </ul>
+          
+        </div>
+</div>
+    </div>
```

Comment:
- [ ] Kindly use `ul` and `li` tags for instead of `div` tags here due to SEO purpose. You can implement them as shown below 👍

```
<ul>
  <li>
    <h3>Languages<h3>
    <ul>
      <li>JavaScript</li>
      <li>HTML</li>
      <li>CSS</li>
    </ul>
  </li>
    <li>
    <h3>Frameworks<h3>
    <ul>
      <li>Ruby on Rails</li>
      <li>CapyBara</li>
      <li>RSpec</li>
      <li>Selenium</li>
      <li>Bootstrap</li>
    </ul>
  </li>
    <li>
    <h3>Skills<h3>
    <ul>
      <li>CodeKit</li>
      <li>Codepen</li>
      <li>Github</li>
      <li>Gitlab</li>
      <li>Terminal</li>
    </ul>
  </li>
<ul>

```